src/mpid/ch4/shm/posix/posix_coll.h

```diff
@@ -101,6 +101,8 @@ static inline int MPIDI_POSIX_mpi_bcast(void *buffer, int count, MPI_Datatype da
     for (i = 0; MPIR_CVAR_BCAST_POSIX_INTRA_ALGORITHM[i] != MPIR_CVAR_ENUM_NULL; i++) {
         switch (MPIR_CVAR_BCAST_POSIX_INTRA_ALGORITHM[i]) {
             case MPIR_CVAR_BCAST_POSIX_INTRA_ALGORITHM_release_gather:
+                if (MPIR_ThreadInfo.isThreaded)
```

Comment:
There is typo in commit msg. `reduce_scatter` -> `release_gather`

Reply:
Fixed.  Thanks for catching.